### DISCUSSION TITLE
Discord for ArchPOWER

### DISCUSSION THREAD

**Discussion by kth5** [kth5/archpower#90] [Announcements]
It was time to bite the bullet.

There is now a Discord server for more real-time discussion of ArchPOWER.

https://discord.gg/HntKjSTrVe

The ins and outs are still being worked on so please bare with us.

We'll find a solution for those unable to connect to Discord using big-endian machines (my apologies if you do).

(edit: link has been refreshed)

**Comment by vasi** [comment]
Discord says that the link is invalid. Can you generate a new one?

**Reply by kth5** [reply]
Could be I didn't hit the "never expire" option and it was set to just 7 days. Link has now been updated.

Thanks for the heads up! :)
